src/com/utils/readers/PropertiesReader.java

```diff
@@ -46,6 +46,29 @@ public String getProperty(String key) {
                 case CHARACTER:
                     value = prop.getProperty(CONFIG.CHARACTER.name());
```

Comment:
please use a map or something else to replace this switch did you notice that is the same line just with different parameters?